ee/tasks/subscriptions/__init__.py

```diff
@@ -9,7 +9,6 @@
 from ee.tasks.subscriptions.email_subscriptions import send_email_subscription_report
 from ee.tasks.subscriptions.slack_subscriptions import send_slack_subscription_report
 from ee.tasks.subscriptions.subscription_utils import generate_assets
-from posthog import settings
```

Comment:
Is this intentionally removed here?

Reply:
Yup, looks like this was merged to `master` without passing `ruff` checks and now I weren't able to merge this branch with master without resolving the duplication :/